components/DropdownLight/DropdownLight.unit.test.jsx

```diff
@@ -19,31 +19,32 @@ describe('<DropdownLight />', () => {
   it('should return a value from item, using a object items', () => {
     render(<DropdownLight items={itemsObjectMock} />);
 
-    const dropdown = screen.getByRole('combobox');
-    userEvent.selectOptions(dropdown, 'Lemon');
+    const dropdown = screen.getByRole('button');
+    userEvent.click(dropdown);
 
-    const lemonItem = itemsObjectMock[0].value.toString();
+    const optionItem = screen.getByText('Lemon');
+    userEvent.click(optionItem);
 
-    expect(dropdown.value).toEqual(lemonItem);
+    const input = Number(screen.getByLabelText('selecione uma opcao').value);
+
+    const lemonItem = itemsObjectMock[0].value;
+
+    expect(input).toEqual(lemonItem);
   });
 
   it('should return a value from item, using a string items', () => {
     render(<DropdownLight items={itemsStringMock} />);
 
-    const dropdown = screen.getByRole('combobox');
-    userEvent.selectOptions(dropdown, 'Lemon');
+    const dropdown = screen.getByRole('button');
+    userEvent.click(dropdown);
 
-    const lemonItem = itemsStringMock[0];
+    const optionItem = screen.getByText('Lemon');
+    userEvent.click(optionItem);
 
-    expect(dropdown.value).toEqual(lemonItem);
-  });
+    const input = screen.getByLabelText('selecione uma opcao').value;
 
-  it('should return default value with [Select an option]', () => {
-    render(<DropdownLight items={itemsStringMock} />);
-
-    const dropdown = screen.getByRole('combobox');
-    const select = screen.getByText('Select an option');
+    const lemonItem = itemsStringMock[0];
 
-    expect(dropdown.value).toEqual(select.value);
+    expect(input).toEqual(lemonItem);
   });
 });
```

Comment:
How about of a new test, `should render the list options when button is clicked`?